frontend/components/exercise/AudioPlayer.tsx

```diff
@@ -1,16 +1,11 @@
 import { useState, useEffect, useRef } from "react";
-import AudioButton from "./AudioButton";
```

Comment:
instead of rewriting the whole audiobutton, is there any way you can adapt the existing audio button which already i believe has custom sizing, to add custom bg color? better modularity then. may need to change ImageCard to add that bg color is white unless. you make it the default. again take a look at what is being used and try to edit rather than write your own